refactor: optimization chapter (#457)
mlr-org/mlr3book: book/basics.qmd

`r mlr3` offers a number of different [measures](#measure) to quantify how well a model performs based on the difference between predicted and actual values.
Usually, this [measure](#measure) is a numeric score.

> Splitting data into training and test sets, building a model, and evaluating it can be repeated several times, resampling different training and test sets from the original data each time (@sec-resampling).
> Multiple resampling iterations allow us to get a better and less biased generalizable performance estimate for a particular type of model.

Reviewer: ```suggestion
Splitting data into training and test sets, building a model, and evaluating it can be repeated several times, resampling dsfdfgdfdfg    particular type of model.
```
